packages/compass-telemetry/src/telemetry-events.ts

```diff
@@ -3158,6 +3278,14 @@ export type TelemetryEvent =
   | AiResponseGeneratedEvent
   | ApplicationLaunchedEvent
   | AtlasLinkClickedEvent
+  | AtlasSkillsAggregationCtaClickedEvent
+  | AtlasSkillsDocumentsCtaClickedEvent
```

Comment:
It seems like you're only using the one on the Documents tab. Do we need all the others?